solr/solrj/src/test/org/apache/solr/client/solrj/request/SolrPingTest.java

```diff
@@ -32,17 +34,19 @@ public class SolrPingTest extends EmbeddedSolrServerTestBase {
   @BeforeClass
   public static void beforeClass() throws Exception {
     File testHome = createTempDir().toFile();
-    FileUtils.copyDirectory(getFile("solrj/solr"), testHome);
-    initCore("solrconfig.xml", "schema.xml", testHome.getAbsolutePath(), "collection1");
+    FileUtils.copyDirectory(SolrTestCaseJ4.getFile("solrj/solr"), testHome);
+
+    SolrTestCaseJ4.newRandomConfig();
+    // TODO NO COMMIT
+    solrClientTestRule.build().setSolrHome(testHome.toPath()).setSchemaFile("schema.xml").init();
```

Comment:
We specify this as exactly `nocommit` (no space).

Any way, I think you are now in a position to fix it by simply removing the needless schema reference.